index.html

```diff
@@ -150,7 +150,31 @@ <h6>languaages</h6>
                 </li>
                  </ul>
             </section>
+            <section class="contact">
+              <h1 class="me">Contact me</h1>
+              <p>If you have an application you
+                are interested in developing, a
+                feature that you need built or aproject that needs coding. 
+                
+                I’d love to help with it!</p>
+                <form class="contact_form">
```

Comment:
- [ ] Great progress on your contact form! 👍🏿 However, I've noticed that there are some crucial implementation aspects missing. According to the project requirements, your form should use the POST method but currently there is no method attribute in your form, i kindly suggest you implement it POST method.
- Using the POST method for form submissions is crucial for maintaining the security and integrity of user data. When a form is submitted via the GET method, the data is appended to the URL as query parameters, which can be visible to anyone who has access to the browser's history or network logs. This exposes sensitive information, such as user credentials or other personal data, to potential security risks.
- [ ] In addition, I noticed that the form action is empty. It is important to have a form action specified in order to properly handle and process the form submissions. According to the project requirements, you are expected to implement a form action using [Formspree](https://formspree.io/html/).

- Formspree is a service that simplifies form handling by providing a backend infrastructure to process and forward form submissions via email. By integrating Formspree into your contact form, you can ensure that the form data is securely collected and sent to the designated email address.